lib/Star/Kata/Infrastructure/InMemoryInfrastructure.php

```diff
@@ -0,0 +1,54 @@
+<?php
+/**
+ * This file is part of the phpkata project.
+ *
+ * (c) Yannick Voyer (http://github.com/yvoyer)
+ */
+
+namespace Star\Kata\Infrastructure;
+
+use Star\Kata\KataDomain\KataService;
+use Star\Kata\Model\KataCollection;
+use Star\Kata\Model\KataRepository;
+
+/**
+ * Class InMemoryInfrastructure
+ *
+ * @author  Yannick Voyer (http://github.com/yvoyer)
+ *
+ * @package Star\Kata\Infrastructure
+ */
+final class InMemoryInfrastructure implements KataInfrastructure
```

Comment:
`InMemoryInfrastructure` doesn't really make it clear what this class's purpose is.. what it is trying to do?